src/util/block_tools.py

```diff
@@ -72,6 +72,8 @@
 
 test_constants = DEFAULT_CONSTANTS.replace(
     **{
+        "MIN_PLOT_SIZE": 18,
```

Comment:
Does 18 really get used anywhere? I thought 22 was the smallest?

Reply:
This was what it's always been. I think we changed chia plots to use 22 or higher because the small ones were inconsistent in plotting.